scripts/inner/kubestellar

```diff
@@ -19,12 +19,20 @@
 #           (a) 1 kcp workspace: edge service provider workspace (espw)
 #           (b) 3 kubestellar controllers: kubestellar-where-resolver, mailbox-controller and placement-translator
 
-# Assumption: kcp server is running.
+# Pre-req: the space manager is running and there is a space provider
+# named "default" and that is the one to use.
+
+# Pre-req: `kubectl` with no flags will access the Space Management
+# API Space (SMAS) with high privilege.
```

Comment:
You're going to add the Space Management API Space to the terminology page?

Reply:
We discussed this in Slack a while ago and agreed on this term.
Yes, it should go in the glossary.
I put an introduction to this term in the introductory material in commands.md.